packaging/ios/Subsurface-mobile.pro

```diff
@@ -28,6 +28,7 @@ SOURCES += ../../subsurface-mobile-main.cpp \
 	../../core/divesitehelpers.cpp \
 	../../core/errorhelper.c \
 	../../core/exif.cpp \
+	../../core/exportfuncs.cpp \
```

Comment:
this also should be part of the one commit that moves the code around

Reply:
This has really nothing to do with moving the code around, it is about making the new code available to qmlmanager, so you could argue it should be part of the qmlmanager commit.

I can see however that this PR makes changes 2 times to the file, and that is not needed so corrected.